### ISSUE TITLE
DNS based discovery not configurable

### ISSUE THREAD

**Issue by JorritSalverda** [pires/docker-elasticsearch-kubernetes#31]
Since commit https://github.com/pires/docker-elasticsearch-kubernetes/commit/d1907dad5d6985c60848b16bdd7bb8af53c78954 it's no longer possible to automatically discover the master nodes in your cluster if the service has a different name. I run multiple ES clusters with different names inside the same namespace.

You used to be able to use the following environment variable:

```
service: ${DISCOVERY_SERVICE}
```

But now the following hardcoded name is used:

```
ping.unicast.hosts: elasticsearch-discovery
```

Is it possible to either revert this change or make sure it's configurable via the same envvar again? Thx

**Comment by JorritSalverda** [comment]
Hm, I see you already did a commit to fix this after 5.1.2. Any chance you can push an updated 5.1.2_1 or something like it? Thx

**Comment by pires** [comment]
Yes.
